library/spdm_responder_lib/libspdm_rsp_algorithms.c

```diff
@@ -631,12 +801,46 @@ libspdm_return_t libspdm_get_response_algorithms(libspdm_context_t *spdm_context
         }
     }
 
+    /* if both PQC and tradition are enabled, choose PQC and disable tradition */
```

Comment:
@jyao I don't find any wording about priority between pqc and traditional algorithms in SPDM1.4-WIP70 spec. Just out of curiosity where the does principle come from?

Reply:
It is implementation choice.

The purpose to use 1.4 is to enable PQC. If no PQC requirement, there is no need to use 1.4.